### DISCUSSION TITLE
how to change xml to html if i use allure report

### DISCUSSION THREAD

**Discussion by garyzhang21** [webdriverio/webdriverio#7614] [Q&A] [answered]
how to change xml to html if i use allure report?

**Comment by christian-bromann** [comment, marked as answer]
See https://webdriver.io/docs/allure-reporter#displaying-the-report

**Reply by garyzhang21** [reply]
yes,  i did it as the docs said. but i can generate successfully xml. but have no ideas to be html... 
![image](https://user-images.githubusercontent.com/32188719/139014464-192849f5-8c46-4173-9dae-820b5d4d5f1d.png)

**Reply by christian-bromann** [reply]
I don't understand, you can or can not create XML and/or HTML? Please elaborate.

**Reply by garyzhang21** [reply]
i can create xml, but i want to create html. What should i do?

**Reply by christian-bromann** [reply]
The above should work. If not, provide a reproducible example, I am happy to take a look.

**Reply by garyzhang21** [reply]
i just did it as https://webdriver.io/docs/allure-reporter#displaying-the-report.
and the first is reporters: [
        [video, {
          saveAllVideos: true,       // If true, also saves videos for successful test cases
          videoSlowdownMultiplier: 3, // Higher to get slower videos, lower for faster videos [Value 1-100]
        }],
        ['allure', {
          outputDir: './Results',
          disableWebdriverStepsReporting: true,
          disableWebdriverScreenshotsReporting: true,
        }],
      ],
and then write the coComplete function as picture. and then run my test. 
and after finish the test, i can get two files. one is xml file. and one is the video.
and my question is why i can not get the .html file? do we have any command to get the .html file, not .xml file?
can i directly to get a .html file?

**Reply by christian-bromann** [reply]
Sorry @garyzhang21 I am not able to reproduce this, you might oversee an error. I created a new project (https://gist.github.com/christian-bromann/c5afa511fd828d79be856c1366c00766) and found my html files in the `allure-report` folder.

**Reply by garyzhang21** [reply]
let me check! anyway. thanks for your helping.

**Reply by garyzhang21** [reply]
btw, from allure-report folder.  i can found the html files. but when i open it, shows like the picture.
![image](https://user-images.githubusercontent.com/32188719/139032880-e5afb1b8-8a66-45db-8774-63c89bb13eec.png)

**Reply by garyzhang21** [reply]
that's interesting.
if i use vscode to open the html file. it will be the picture.
but if i use idea to open the html file. i can get my report.
though i am not sure the reason
but anyway, success..

**Reply by BorisOsipov** [reply]
@garyzhang21 it is expected behavior. to open allure html you have to run some webserver to serve report html. you cannot just open report file from folder.

**Comment by Jyoti2190-Test** [comment]
Hi Gary  Are u using allure open to view the report. I am not able to see reports neither from html in allure report folder nor from allure open command

**Reply by garyzhang21** [reply]
Hi sorry for the late reply.
Can you try use idea to open your html?
